baseline.yaml

```
@@ -192,6 +192,34 @@ criteria:
     scorecard_probe:
       - # TODO
 
+  - id: OSPS-AC-08
+    maturity_level: 1
```

```suggestion
    maturity_level: 3
```

This criterion cannot be met by 1-person projects. The [vast majority of OSS projects are 1-person projects (per TAC issue 101)](https://github.com/ossf/tac/issues/101), and this is not under the control of maintainers. Maintainers can take steps to make it *easier* to join a project, and they should, but maintainers don't decide how others spend their time.

For projects with 2-3 developers who are not doing this as part of their daytime work, requiring second review can greatly slow development. You're waiting for the other guy to be available, which may be next month or maybe the month after that.

Don't get me wrong, I'm a huge fan of the advantages of review before merging. But while enforcing it easy, that doesn't make extra people available for most projects.

If the intent is that this baseline _cannot_ be applied by the vast majority of OSS projects, then that needs to be made clearer, e.g., "this baseline is only intended for larger projects with multiple funded developers" or whatever is the intent. We should make that clearer if that's the intent.

---

@david-a-wheeler Can we split the difference and say `lv2`?

I've been on a couple 2-3 person projects including one right now where we enforce this and—yeah— it slows things down, but it dramatically reduces the defects we introduce.